andela/Ah-frontend-xmen
# #162725218 Users should be able to signup

##### What does this Pr do?
- Adds an implementation to enable users to create new accounts on the authors' heaven platform.

##### Description of the task to be implemented.
- Create  a signup form to allow users to enter the details for their new accounts
- Create a component to handle the signup process by sending user information to the backend API.
- Connect the component to the global store.
- When users enter incorrect details, they should be presented with descriptive validation errors.

#### How can this be manually tested?
- Fetch this branch to your local machine by running `git fetch origin ft-users-should-signup-162725218` in your terminal.
- Checkout to this branch by running ` git checkout ft-users-should-signup-162725218` in your terminal.
- Install project dependencies by running `yarn install or just yarn`
- Start the development server by running `yarn start` in your terminal.
- Navigate to the signup route by clicking on the `Get Started` link in the browser.
- Enter your details into the form and click the signup button.
- On success, you should receive an email inviting you to verify your email.

#### What are the related pivotal stories?
[162725218](https://www.pivotaltracker.com/story/show/162725218)
